### PR TITLE
SRS backend unit testcases

### PR DESCRIPTION
**Pull request by Yagnik56**
This PR contains backend unit testcases for SRS feature

### CODE REVIEW

File: backend/packages/Upgrade/test/unit/controllers/StratificationController.test.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  afterAll(() => {
+    Container.reset();
+  });
+
+  test('Get request for /api/stratification', async (done) => {
+    await request(app).get('/api/stratification').expect('Content-Type', /json/).expect(200);
+    done();
+  });
+
+  const factorName = 'factor1';
```
Comment (VivekFitkariwala) [inline backend/packages/Upgrade/test/unit/controllers/StratificationController.test.ts:33]: These code should be moved inside the testcase where it is used.

File: backend/packages/Upgrade/test/unit/controllers/StratificationController.test.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      file: fs.readFileSync(tempCsvFilePath, 'utf-8'),
+    },
+  ];
+
+  test('Post request for /api/stratification', async (done) => {
+    await request(app).post(`/api/stratification`).send(requestBody).expect('Content-Type', /json/).expect(200);
+    done();
+  });
+
+  fs.unlinkSync(tempCsvFilePath);
```
Comment (VivekFitkariwala) [inline backend/packages/Upgrade/test/unit/controllers/StratificationController.test.ts:58]: This code should be moved inside the testcase

**Review by VivekFitkariwala — CHANGES_REQUESTED** [review]
Small change

File: backend/packages/Upgrade/test/unit/controllers/StratificationController.test.ts (hunk 2 above)
Comment (Yagnik56) [reply backend/packages/Upgrade/test/unit/controllers/StratificationController.test.ts]: resolved

File: backend/packages/Upgrade/test/unit/controllers/StratificationController.test.ts (hunk 1 above)
Comment (Yagnik56) [reply backend/packages/Upgrade/test/unit/controllers/StratificationController.test.ts:33]: resolved

**Review by VivekFitkariwala — APPROVED** [review]

**Pull request merged**